brave/brave-core
# Reddit inline tips link on post menu will now appear

Fixes brave/brave-browser#4915

brave-ui: https://github.com/brave/brave-ui/pull/498 https://github.com/brave/brave-ui/pull/501

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

1. Visit reddit.com with Rewards enabled and ensure that tip link shows on reddit posts' flat menu
2. Go to rewards settings->tip settings and uncheck the box for 'reddit' and ensure that inline tip links do not show on reddit.
3. Ensure tipping banner is correct when using reddit inline tipping link.


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
